README.md

```diff
@@ -20,6 +20,7 @@ limitations under the License.
 
 - [Running TGI on Gaudi](#running-tgi-on-gaudi)
 - [Adjusting TGI parameters](#adjusting-tgi-parameters)
+- [Running TGI with fp8 precision](#running-tgi-with-fp8-precision)
```

Comment:
Please change to "FP8" everywhere, like in https://github.com/huggingface/optimum-habana/tree/main/examples/text-generation#running-with-fp8

Reply:
Fixed